### PR TITLE
Update Polish translation for Tour of Scala

### PR DESCRIPTION
**Pull request by kamilduda**
Updated Polish translation for: tour-of-scala, basics, unified-types.
Made tuples visible on Contents menu of English version.

### CODE REVIEW

File: _pl/tour/tour-of-scala.md [hunk 1]
```diff
@@ -7,41 +7,68 @@ discourse: false
...
+* [górne](upper-type-bounds.html) oraz [dolne](lower-type-bounds.html) ograniczenia typów,
+* [klasy zagnieżdżone](inner-classes.html) i [typy abstrakcyjne](abstract-types.html) jako elementy obiektów,
+* [typy złożone](compound-types.html),
+* [jawnie typowane samoreferencje](self-types.html),
+* [parametry domniemane](implicit-parameters.html) i [konwersje niejawne](implicit-conversions.html),
+* [metody polimorficzne](polymorphic-methods.html).
 
-[Mechanizm lokalnej inferencji typów](type-inference.html) sprawia, że nie jest konieczne podawanie nadmiarowych informacji o typach w programie. W połączeniu te funkcje języka pozwalają na bezpiecznie typowane ponowne wykorzystanie programistycznych abstrakcji.
+[Mechanizm lokalnej inferencji typów](type-inference.html) sprawia, że nie jest konieczne podawanie nadmiarowych informacji o typach w programie.
+Wszystkie te funkcje języka pozwalają na bezpiecznie reużywanie programistycznych abstrakcji oraz rozwijanie oprogramowania z bezpieczeństwem dla typów.
```
Comment (jstefanska) [inline _pl/tour/tour-of-scala.md]:
```suggestion
Wszystkie te funkcje języka pozwalają na bezpieczne ponowne użycie programistycznych abstrakcji oraz rozwijanie oprogramowania z bezpieczeństwem dla typów.
```

File: _pl/tour/tour-of-scala.md [hunk 2]
```diff
@@ -7,41 +7,68 @@ discourse: false
...
 
 ## Scala jest rozszerzalna ##
-W praktyce rozwiązania specyficzne dla domeny wymagają odpowiednich rozszerzeń języka. Scala dostarcza unikalne mechanizmy, dzięki którym można łatwo dodawać nowe konstrukcje do języka w postaci bibliotek:
+W praktyce, rozwiązania specyficzne dla domeny często wymagają odpowiednich, również specyficznych domenowo, rozszerzeń języka.
```
Comment (jstefanska) [inline _pl/tour/tour-of-scala.md]:
```suggestion
W praktyce rozwiązania specyficzne dla domeny często wymagają odpowiednich, również specyficznych domenowo, rozszerzeń języka.
```

File: _pl/tour/tour-of-scala.md [hunk 3]
```diff
@@ -7,41 +7,68 @@ discourse: false
...
+* [Interpolacja łańcuchów znaków](/overviews/core/string-interpolation.html) jest rozszerzalna o niestandardowe interpolatory użytkownika.
+
+## Scala współdziała z językami JVM
+Scala jest zaprojektowana tak, aby jak najlepiej współpracować z popularnym środowiskiem uruchomieniowym Java Runtime Environment (JRE).
+W szczególności interakcja z językiem Java jest tak płynna, jak to tylko jest możliwe.
+Nowsze funkcje języka Java takie jak interfejsy funkcyjne (SAM), [funkcje anonimowe (lambda)](higher-order-functions.html), [adnotacje](annotations.html) oraz [typy generyczne](generic-classes.html) posiadają swoje bezpośrednie odwzorowania w języku Scala.
 
-* każda metoda może być używana jako [operator infiksowy lub prefiksowy](operators.html)
-* [domknięcia są konstruowane automatycznie zależnie od wymaganego typu](automatic-closures.html)
+Unikalne dla Scali funkcje, które nie mają odwzorowanie w Javie, jak na przykład [domyślne wartości parametrów](default-parameter-values.html) oraz [parametry nazwane](named-arguments.html), są kompilowane tak, aby zachować jak największą zgodność z Javą.
```
Comment (jstefanska) [inline _pl/tour/tour-of-scala.md]:
```suggestion
Unikalne dla Scali funkcje, które nie mają odwzorowania w Javie, jak na przykład [domyślne wartości parametrów](default-parameter-values.html) oraz [parametry nazwane](named-arguments.html), są kompilowane tak, aby zachować jak największą zgodność z Javą.
```

File: _pl/tour/tour-of-scala.md [hunk 4]
```diff
@@ -7,41 +7,68 @@ discourse: false
...
+
+W większości przypadków można to uzyskać bez potrzeby używania technik meta-programowania takich jak np. makra.
+Oto kilka przykładów:
+
+* [Klasy domniemane](http://docs.scala-lang.org/overviews/core/implicit-classes.html) pozwalają na rozszerzanie już istniejących klas o nowe metody,
+* [Interpolacja łańcuchów znaków](/overviews/core/string-interpolation.html) jest rozszerzalna o niestandardowe interpolatory użytkownika.
+
+## Scala współdziała z językami JVM
+Scala jest zaprojektowana tak, aby jak najlepiej współpracować z popularnym środowiskiem uruchomieniowym Java Runtime Environment (JRE).
+W szczególności interakcja z językiem Java jest tak płynna, jak to tylko jest możliwe.
```
Comment (jstefanska) [inline _pl/tour/tour-of-scala.md]:
```suggestion
W szczególności interakcja z językiem Java jest tak płynna, jak tylko jest to możliwe.
```

File: _pl/tour/unified-types.md [hunk 5]
```diff
@@ -6,44 +6,101 @@ discourse: false
...
-num: 2
+num: 3
 language: pl
 next-page: classes
-previous-page: tour-of-scala
+previous-page: basics
 ---
 
-W przeciwieństwie do Javy wszystkie wartości w Scali są obiektami (wliczając w to wartości numeryczne i funkcje). Ponieważ Scala bazuje na klasach, wszystkie wartości są instancjami klasy. Można zatem powiedzieć, że Scala posiada zunifikowany system typów. Poniższy diagram ilustruje hierarchię klas Scali:
+W Scali wszystko wartości mają określony typ, włączając w to wartości numeryczne i funkcje.
```
Comment (jstefanska) [inline _pl/tour/unified-types.md]:
```suggestion
W Scali wszystkie wartości mają określony typ, włączając w to wartości numeryczne i funkcje.
```

File: _pl/tour/unified-types.md [hunk 6]
```diff
@@ -6,44 +6,101 @@ discourse: false
...
+## Hierarchia Typów Scali ##
 
-Klasa bazowa dla wszystkich klas `scala.Any` posiada dwie bezpośrednie klasy pochodne: `scala.AnyVal` oraz `scala.AnyRef`, które reprezentują dwie różne rodziny klas: klasy wartości oraz klasy referencji. Klasy wartości są predefiniowane i odpowiadają one typom prymitywnym z języków takich jak Java. Wszystkie inne klasy definiują typy referencyjne. Klasy zdefiniowane przez użytkownika są domyślnie typami referencyjnymi, tzn. są one zawsze podtypem klasy `scala.AnyRef`. W kontekście maszyny wirtualnej Javy `scala.AnyRef` odpowiada typowi `java.lang.Object`. Powyższy diagram ilustruje także konwersje implicit pomiędzy klasami wartości.
+Typem bazowym dla wszystkich klas jest `Any`, jest on też nazywany typem górnym (top type).
+Definiuje on uniwersalne metody takie jak `equals`, `hashCode` oraz `toString`.
+`Any` posiada dwa bezpośrednie podtypy: `AnyVal` i `AnyRef`.
 
-Poniższy przykład pokazuje, że liczby, znaki, wartości logiczne oraz funkcje są obiektami:
+`AnyVal` reprezentuje typy wartości.
+Żadne z tych typów nie może przyjąć wartości `null`.
```
Comment (jstefanska) [inline _pl/tour/unified-types.md]:
```suggestion
Żaden z tych typów nie może przyjąć wartości `null`.
```

File: _pl/tour/unified-types.md [hunk 7]
```diff
@@ -6,44 +6,101 @@ discourse: false
...
+Istnieje dziewięć predefiniowanych typów wartości: `Double`, `Float`, `Long`, `Int`, `Short`, `Byte`, `Char`, `Unit` oraz `Boolean`.
+`Unit` to typ wartości, która nie niesie ze sobą żadnej znaczącej informacji.
+Istnieje dokładnie jedna instancja typu `Unit` i jest zdefiniowana dosłownie jako: `()`.
+Wszystkie funkcje muszą coś zwracać, dlatego w niektórych przypadkach trzeba użyć `Unit` do oznaczenia zwracanego typu.
 
+`AnyRef` reprezentuje typy referencyjne.
+Przez referencje mamy w tym przypadku na myśli wskaźniki do innych obiektów.
+Wszystkie typy nie będące wartościami są zdefiniowane jako typy referencyjne.
```
Comment (jstefanska) [inline _pl/tour/unified-types.md]:
```suggestion
Wszystkie typy niebędące wartościami są zdefiniowane jako typy referencyjne.
```

File: _pl/tour/basics.md [hunk 8]
```diff
@@ -1,9 +1,350 @@
...
 ---
+
+Na tej stronie omówimy podstawy języka Scala.
+
+## Uruchamianie Scali w przeglądarce
+
+Dzięki ScalaFiddle możesz uruchomić Scale w swojej przeglądarce.
+
+1. Przejdź do [https://scalafiddle.io](https://scalafiddle.io).
+2. Wklej kod `println("Hello, world!")` w lewej części okna.
```
[the remark this answers is not in the document]
Comment (kamilduda) [reply _pl/tour/basics.md]: Sprawdź https://scalafiddle.io/ - jest to raczej lewa strona.
Może w tym miejscu "w polu po lewej strone" ?

**Comment by jstefanska** [comment]
Look all right to me now 😃

**Comment by SethTisue** [comment]
can you rebase to see if it fixes the CI failure...?

**Comment by kamilduda** [comment]
done

**Comment by SethTisue** [comment]
thank you @kamilduda and @jstefanska!

**Pull request merged**